manifests/params.pp

```diff
@@ -91,6 +91,7 @@
   $conf_d                     = "${confdir}/conf.d"
   $config_file                = "${confdir}/unbound.conf"
   $control_enable             = false
+  $control_setup_path         = '/usr/sbin/unbound-control-setup'
```

Comment:
exactly…
and that way it can easily be expanded to new platforms with different paths.